### PR TITLE
release: 1.6.1

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

Maybe we can include some other fix and then release it

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,34 +9,33 @@ New entries must be placed in a section entitled `Unreleased`.
 Read
 our [guidelines for writing a good changelog entry](https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#changelog).
 
-## Unreleased
+## 1.6.1 (2024-11-08)
```
Comment (SuperchupuDev) [inline CHANGELOG.md:12]: remember to set this to the correct date before merging

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,34 +9,33 @@ New entries must be placed in a section entitled `Unreleased`.
...
 
 ### Analyzer
 
 ### CLI
 
-### Configuration
+- Fix [#2008](https://github.com/biomejs/biome/issues/2008), where `biome.jsonc` wasn't correctly loaded in some circumstances. Contributed by @Sec-ant
```
Comment (Sec-ant) [inline CHANGELOG.md:18]: Feel free to overwrite mine in #2051.

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,34 +9,33 @@ New entries must be placed in a section entitled `Unreleased`.
...
 
-### Editors
+### Configuration
 
-### Formatter
+#### Bug fixes
 
-### JavaScript APIs
+- Fix [#2099](https://github.com/biomejs/biome/issues/2009), where `json.formatter.trailingCommas` was correctly computed. Contributed by @Sec-ant
```
Comment (Sec-ant) [inline CHANGELOG.md:24]: Feel free to overwrite mine in #2051.

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,34 +9,33 @@ New entries must be placed in a section entitled `Unreleased`.
...
-
-#### Enhamcements
+#### Bugs fixes
 
-#### Bug fixes
+-  Fix [#2011](https://github.com/biomejs/biome/issues/2011), where [`useJsxKeyInIterables`](https://biomejs.dev/linter/rules/use-jsx-key-in-iterable/) thrown a false positive in some cases:
+  ```jsx
+  (<h1>{data.map(c => (<h1 key={c}></h1>))}</h1>)
+  ```
+  Contributed by @Sec-ant
```
Comment (Sec-ant) [inline CHANGELOG.md:34]: Feel free to overwrite mine in #2051.

**Review by Sec-ant — COMMENTED** [review]
Oops, I also added a few change logs in #2051. Feel free to overwrite them.

**Comment by ematipico** [comment]
@Sec-ant I kept your original entries :)

**Pull request merged**